src/Nethermind/Nethermind.Synchronization.Test/ParallelSync/MultiSyncModeSelectorFastSyncTests.cs

```diff
@@ -474,6 +474,16 @@ public void Does_not_move_back_to_state_sync_mistakenly_when_in_full_sync_becaus
                 .TheSyncModeShouldBe(GetExpectationsIfNeedToWaitForHeaders(SyncMode.Full | SyncMode.FastHeaders));
         }
 
+        [Test]
+        public void When_state_sync_does_not_finished_then_sync_mode_should_be_full()
```

Comment:
Do you mean "then_sync_mode_should_not_be_full"?

Reply:
yes, thx